### ISSUE TITLE
Adjust the proper Temperature for softmax function.

### ISSUE THREAD

**Issue by srvkmr130** [Void-Network/AUTO-CAR#1]
> def select_action(self, state):
>         probs = F.softmax(self.model(Variable(state, volatile = True))*100) # T=100
>         action = probs.multinomial(num_samples=1)
>         return action.data[0,0]

The softmax function gives better result when the temperature is fixed appropriately.
